File: CS/CustomChartCursor/Form1.cs
#region #usings
using System;
using System.Windows.Forms;
using DevExpress.XtraCharts;
// ...
#endregion #usings

namespace CustomChartCursor {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

#region #code
private void chartControl1_QueryCursor(object sender, QueryCursorEventArgs e) {
    if (e.CursorType == CursorType.Hand || e.CursorType == CursorType.Grab) {
        e.Cursor = Cursors.Default;
    }
}
#endregion #code

    }
}

File: VB/CustomChartCursor/Form1.vb
Imports Microsoft.VisualBasic
#Region "#usings"
Imports System
Imports System.Windows.Forms
Imports DevExpress.XtraCharts
' ...
#End Region ' #usings

Namespace CustomChartCursor
    Partial Public Class Form1
        Inherits Form
            Public Sub New()
                InitializeComponent()
            End Sub

#Region "#code"
Private Sub chartControl1_QueryCursor(ByVal sender As Object, _ 
ByVal e As QueryCursorEventArgs) Handles chartControl1.QueryCursor
    If e.CursorType = CursorType.Hand OrElse e.CursorType = CursorType.Grab Then
        e.Cursor = Cursors.Default
    End If
End Sub
#End Region ' #code

    End Class
End Namespace
